star-wars-browser/src/store/peopleData/actions.js

```diff
@@ -0,0 +1,8 @@
+import { createAction } from '@reduxjs/toolkit';
+
+// eslint-disable-next-line import/prefer-default-export
+export const setPeopleData = createAction('peopleData/setPeopleData', (dataProvided) => ({
+  payload: {
+    dataProvided,
```

Comment:
może warto byłoby pozmieniać wszędzie dataProvided na coś lepiej mówiącego co to jest, na przykład people albo peopleList

wszędzie - mam na myśli w payloadach akcji, w storze, reducerach, itd.